src/index.jsx

```diff
@@ -21,22 +21,18 @@ import ProtectedRoute from './common/ProtectedRoute';
 import { AUTH_ROLES } from './common/config';
 
 const { ADMIN_ROLE, USER_ROLE } = AUTH_ROLES;
-// TODO: add protected routes -- use current few as examples
-// specify component to render if access allowed, redirectPath if access denied
-// use roles from AUTH_ROLES; add to roles list
+// TODO: add routes for access-denied and not-found pages
 
 ReactDOM.render(
   <React.StrictMode>
     <CookiesProvider>
       <Router>
         <Routes>
           <Route path="/" exact element={<LoginView />} />
-          <Route path="/sites/create/" exact render={() => window.location.replace('/sites')} />
-          <Route path="/sites/create/:areaId" exact element={<SitesCreateView />} />
           <Route path="/login" exact element={<LoginView />} />
           <Route path="/login/reset-password" exact element={<LoginResetPasswordView />} />
-          <Route path="/master-teachers" exact element={<TeacherView />} />
-          <Route path="/master-teachers/export-data" exact element={<TeachersExportDataView />} />
+          {/* <Route path="/master-teachers" exact element={<TeacherView />} /> */}
```

Comment:
I wasn't sure if these were actual pages because it didn't look like there was anything on them. Let me know if I should leave and uncomment or remove them.